### PR TITLE
Blake stanger assignment2

### PR DESCRIPTION
**Pull request by blakestanger**
-Fixed file structure (./methods-spring-2020/problems/*type*/*problem-name*/*language*/*yourname*/)

-wrapped solution in a function(cold-puter def calculate days)

-removed old bad file structure from remote repository.

### CODE REVIEW

File: problems/kattis/Cold-puter/python/BlakeStanger/Cold-puter.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        elif opt in ("-i", "--ifile"):
+            inputfile = arg
+    print("Input file is "), inputfile
+    calculate_days(inputfile)
+
+
+def calculate_days(inputfile):
+    # opens and parses subzero days.
+    with open(inputfile) as fileobj:
+        vg = fileobj.readline().rstrip('\n')
```
Comment (wyatt-benson) [inline problems/kattis/Cold-puter/python/BlakeStanger/Cold-puter.py:28]: What is vg for? It appears identical to c but is never used.

**Review by wyatt-benson — APPROVED** [review]
+1 
Possible extra variable in Cold-puter.py but everything else looks great.
Very thorough readme

**Comment by sleddog** [comment]
Remove the changes favorite/* files from this branch.  Keep this PR to only the assignment2 requirements.
Rename the README.txt to README.md
And resolve merge conflicts.

**Review by Jason-Bengtson — APPROVED** [review]
+1

**Comment by blakestanger** [comment]
Hey Devin how would I accomplish this? 
> Remove the changes favorite/* files from this branch. Keep this PR to only the assignment2 requirements.

**Pull request merged**